app/src/main/java/org/xwiki/android/sync/activities/AccountListAdapter.kt

```diff
@@ -0,0 +1,69 @@
+package org.xwiki.android.sync.activities
+
+import android.accounts.Account
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.BaseAdapter
+import android.widget.LinearLayout
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import org.xwiki.android.sync.R
+import org.xwiki.android.sync.utils.AccountClickListener
+
+class AccountListAdapter (
+    private val mContext: Context,
+    private var availableAccounts : Array<Account>,
+    private val listener : AccountClickListener)
+    : BaseAdapter()  {
+
+    override fun getView(position: Int, convertView: View?, parent: ViewGroup?): View {
+        var convertView = convertView
+        val viewHolder: AccountListViewHolder
+
+        if (convertView == null) {
+            val inflater = LayoutInflater.from(mContext)
```

Comment:
@InsanusMokrassar Cannot replace convertView to let coz when we put let to convertView it would become non-null value(convertView?) and the return statement requires convertView (null value)

Reply:
Anyway, you can replace it with also

```kotlin
convertView ?.also {
    …
} ?:also {
    …
}
```

Reply:
Done